client/components/Overview/Overview components/ProductInfo components/Info.jsx

```diff
@@ -1,11 +1,21 @@
 import React from 'react';
+import styled from 'styled-components';
 
-const Info = (props) => (
-  <div id="info">
-    Category
-    Product Title
-    Product Price
-  </div>
-);
+const StyledPInfoContainer = styled.div`
+  background-color: lightcoral;
+`;
+
+const Info = (props) => {
```

Comment:
Instead of accessing all the products could you just pass down the displayedProduct state?